### PR TITLE
Enhance README.md

### PR DESCRIPTION
**Pull request by alexandrefch**
A proposal to improve README

+ Add contributors
+ Change package link from `v2.0.4` to `v2.0.5`
+ Add shieldio badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,28 +34,78 @@ The following sensors are added.
 - (GroundTruth)
 - (TF)
 
-There are several Prefab and Scene files available for testing each sensor.
+**2. UnitySensorRos**
```
Comment (Autumn60) [inline README.md]:
```suggestion
**2. UnitySensorROS**
```

File: README.md [hunk 2]
```diff
@@ -19,28 +34,78 @@ The following sensors are added.
...
+
+This package is responsible to make the link between sensor and ros by serializing sensor raw data and sending to them to ROS using [RosTCPConnector][external-RosTCPConnector-link] package.
```
Comment (Autumn60) [inline README.md]:
```suggestion
This package is responsible to make the link between sensor and ROS by serializing sensor raw data and sending to them to ROS using [ROS-TCP-Connector][external-RosTCPConnector-link] package.
```

File: README.md [hunk 3]
```diff
@@ -19,28 +34,78 @@ The following sensors are added.
...
+To receive the data in ROS take a look at [ROSTCPEndpoint][external-RosTCPEndpoint-link].
```
Comment (Autumn60) [inline README.md]:
```suggestion
To receive the data in ROS take a look at [ROS-TCP-Endpoint][external-RosTCPEndpoint-link].
```

File: README.md [hunk 4]
```diff
@@ -19,28 +34,78 @@ The following sensors are added.
...
+3. Enter the git URL for the desired package.
+    1. For the UnitySensors, enter `https://github.com/Field-Robotics-Japan/UnitySensors.git?path=/Assets/UnitySensors#v2.0.5`.
+    2. For the UnitySensorsROS, enter `https://github.com/Field-Robotics-Japan/UnitySensors.git?path=/Assets/UnitySensorsROS#v2.0.5`.
     __Note: UnitySensorsROS does not contain UnitySensors.__
 4. Click `Add`.
 
-## Documentation
+### Inside ROS workspace
+
+1. Download lastest release of [ROSTCPEndpoint][external-RosTCPEndpoint-release-link].
```
Comment (Autumn60) [inline README.md]:
```suggestion
1. Download lastest release of [ROS-TCP-Endpoint][external-RosTCPEndpoint-release-link].
```

File: README.md [hunk 5]
```diff
@@ -19,28 +34,78 @@ The following sensors are added.
...
+<div align="center">
+
+A huge thank you to everyone who is helping to improve UnitySensors !
+
+[![contributors badge][github-contributors-img]][github-contributors-link]
+
+</div>
+
+## 🔗 Dependencies
+- [RosTCPConnector][external-RosTCPConnector-link] (Appache 2.0 LICENSE)
```
Comment (Autumn60) [inline README.md]:
```suggestion
- [ROS-TCP-Connector][external-RosTCPConnector-link] (Appache 2.0 LICENSE)
```

**Review by Autumn60 — CHANGES_REQUESTED** [review]
This is cool.
I always learn something from your PR.

Could you check my suggestion? 
They are minor corrections of proper nouns.

File: README.md (hunk 2 above)
Comment (Autumn60) [inline README.md]:
```suggestion
This package is responsible to make the link between sensor and ros by serializing sensor raw data and sending to them to ROS using [ROS-TCP-Connector][external-RosTCPConnector-link] package.
```

**Comment by alexandrefch** [comment]
I took your suggestions into account and push a new version

**Review by Autumn60 — APPROVED** [review]
LGTM.
Thanks!!

**Pull request merged**